__tests__/offlineUI.js

```diff
@@ -214,4 +214,81 @@ describe('offline UI', async () => {
 
     // Snapshots are intentionally not compared because the spinner will cause the snapshot to fail regularly
   });
+
+  test('should display "Network interruption occurred. Reconnecting…" status when connection is interrupted', async () => {
+    const { driver } = await setupWebDriver({
+      createDirectLine: options => {
+        const reconnectingDirectLine = window.WebChat.createDirectLine(options);
+
+        return {
+          activity$: reconnectingDirectLine.activity$,
+          postActivity: reconnectingDirectLine.postActivity.bind(reconnectingDirectLine),
+
+          connectionStatus$: new Observable(observer => {
+            const subscription = reconnectingDirectLine.connectionStatus$.subscribe({
+              complete: () => observer.complete(),
+              error: err => observer.error(err),
+              next: connectionStatus => {
+                observer.next(connectionStatus);
+                connectionStatus === 2 && observer.next(1);
+              }
+            });
+
+            return () => subscription.unsubscribe();
+          })
+        };
+      },
+      pingBotOnLoad: false,
+      setup: () => new Promise(resolve => {
+        const scriptElement = document.createElement('script');
+        scriptElement.onload = resolve;
+        scriptElement.setAttribute('src', 'https://unpkg.com/core-js@2.6.3/client/core.min.js');
+
+        document.head.appendChild(scriptElement);
+      })
+    });
+
+    await driver.sleep(600);
+    const base64PNG = await driver.takeScreenshot();
```

Comment:
Is taking a screenshot necessary if they are never compared?

Reply:
The sleep is necessary so that if the screenshot is taken, it doesn't take the screenshot too early.

I left these lines here so that if we do need to compare screenshots, only the compare screenshots line needs to be added. But I see what you mean. What do you think of the following fix? 

```
// Snapshots are intentionally not compared because the spinner will cause the snapshot to fail regularly
// Uncomment the following lines to re-capture snapshot or do a manual comparison

// await driver.sleep(600);
// const base64PNG = await driver.takeScreenshot();
// expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
```